buildSrc/src/main/groovy/edu/wpi/gripgenerator/CodeGenerator.groovy

```diff
@@ -26,8 +26,7 @@ class CodeGenerator extends DefaultTask {
     @TaskAction
     def runAction() {
         //Get the target directory
-        LinkedHashSet destSet = dest;
-        String targetDirectoryString = destSet.getAt(0).toString()
+        String targetDirectoryString = dest;
```

Comment:
It's now being passed a string from the gradle build file instead. Not a list of files